README.md

```diff
@@ -372,12 +396,28 @@ Alternatively if you use the `<our-link>` without child DOM elements then it wil
 
 With this tag helper the child DOM elements inside the `<our-link>` is wrapped with the `<a>` tag
 
+### Making the link optional
+
+You can also add `Fallback` for the link to become optional, if there is no link and there is child content, this content is still rendered, you can also replace the wrapping element by using `FallbackElement`.
+
+```cshtml
+<our-link link="@Model.ctaLink" Fallback="true" FallbackElement="div">
+    <span>some text</span>
+</our-link>
+
+@*No link will result in*@
+<div>
+    <span>some text</span>
+</div>
+```
```

Comment:
```suggestion
```cshtml
<our-link link="@Model.ctaLink" Fallback="true" FallbackElement="div">
    <span>some text</span>
</our-link>

@*No link will result in*@
<div>
    <span>some text</span>
</div>

<our-link link="@Model.ctaLink" Fallback="true">
    <span>some text</span>
</our-link>

@*No link and no fallback HTML tag name specified will result in*@
<span>some text</span>
```
```